Backpack-SwiftUI/Skeleton/Classes/BPKSkeletonStyle.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+extension BPKSkeleton {
+    public enum Style {
+        case `default`, rounded
+    }
+}
+
+internal extension BPKSkeleton.Style {
+    var cornerRadius: CGFloat {
+        switch self {
+        case .rounded:
+            return BPKCornerRadius.sm.value
+        default:
```

Comment:
could this be like this instead?
```swift
case .`default`:
```

Reply:
It's more a nice to have, just in case we add more cases in the future and we dont 'forget' to handle this switch